integration/__tests__/ethereum_scen.js

```diff
@@ -0,0 +1,18 @@
+const { buildScenarios } = require('../util/scenario');
+
+let ethereum_scen_info = {
+  tokens: []
+};
+
+buildScenarios('Ethereum Scenarios', ethereum_scen_info, [
+  {
+    name: "Locker: Lock and Extract",
+    scenario: async ({ chain, zrx, eth, starport }) => {
+      let locker = await eth.__deploy('Locker', [starport.ethAddress()]);
+      await locker.methods.lockAndtransfer().send({ from: eth.root(), value: 0.1e18 });
+      // This should, if all goes well, XXX
```

Comment:
Kept just the contracts since the 'max transfer less fee' is already integrated. This scen is incomplete but going to merge so we don't lose these useful contracts